model/testutil/api.go

```diff
@@ -98,11 +98,14 @@ func SetupAPITestData(testConfig *evergreen.Settings, taskDisplayName string, va
 	}
 
 	// Save the project variables
+	if len(testConfig.Providers.AWS.EC2Keys) == 0 {
+		return nil, errors.New("no EC2 Keys in test config")
+	}
 	projectVars := &model.ProjectVars{
 		Id: project.DisplayName,
 		Vars: map[string]string{
-			"aws_key":    testConfig.Providers.AWS.EC2Key,
-			"aws_secret": testConfig.Providers.AWS.EC2Secret,
+			"aws_key":    testConfig.Providers.AWS.EC2Keys[0].Key,
```

Comment:
Do you know what project vars are used for? Will it always be correct to use the first key/secret?

Reply:
I couldn't find somewhere specific where we use the project variables set here, but current uses I'm aware of for the `aws_key` and `aws_secret` variables should be good with credentials for the region Evergreen finds itself in. The mock for these credentials are in position 0 [here](https://github.com/evergreen-ci/evergreen/pull/2898/files#diff-5da188f17bfe0e26cd9d3e20be0baa2aR25).